pre-commit

```diff
@@ -372,6 +372,27 @@ class RepoTreeTest(TestCase):
                     check_var_srcinfo('replaces', package, PACKAGES, line)
                     check_var_srcinfo('groups', package, GROUPS, line)
 
+    def test_repo_depends_valid(self):
+        pkgs = list(list_packages())
```

Comment:
This looks strange. `list_packages` should be called `iter_packages`.

Reply:
Sort of. `list_packages` has been a generator since its introduction.

Reply:
we can change this function name after this got merged